recipes/plugins_development.ipynb

```diff
@@ -6,11 +6,11 @@
    "source": [
```

Comment:
<p>"For instance, you can trigger a plugin when a labeler clicks on Submit&nbsp;with the&nbsp;on_submit&nbsp;handler" is split on two line</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/876/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
@theodu I'm not sure what the problem is. Can you add some more info?

Reply:
Sorry, i didn't give enough detail. I have the impression that the sentence that I sent in the message above is split 2 lines ? But may be it is only on the NB review app

Reply:
Issue only exists in the review app, when new and old code is shown side-by-side. Closing.